examples/lib/defaultHtmlStepUi.js

```diff
@@ -403,7 +403,9 @@ function DefaultHtmlStepUi(_sequencer, options) {
     if (_sequencer.steps.length - 1 > 1) $('#load-image .insert-step').prop('disabled', false);
     else $('#load-image .insert-step').prop('disabled', true);
 
-    $('div[class*=imgareaselect-]').remove();
+    $(step.imgElement).imgAreaSelect({
```

Comment:
Okay I found something.
https://github.com/publiclab/image-sequencer/blob/423422a61f61f1a4fa9149b9dafee1508bbe350e/examples/lib/defaultHtmlStepUi.js#L83

`step.imgElement` is already a jquery object. You don't need to wrap it in another `$()`

Reply:
we need to do it otherwise we have an error step.imgElement.imgAreaSelect is not a function